baybe/utils/plotting.py

```diff
@@ -0,0 +1,56 @@
+"""Plotting utilities."""
+
+import json
+import os
+import warnings
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
+
+
+def create_plots(data: pd.DataFrame, path: Path, base_name: str, **kwargs) -> None:
+    """Create plots from a given data frame and save them as a svg file.
+
+    The plots will be saved in the location specified by ``path``.
+    The attribute ``base_name`` is used to define the name of the outputs.
+
+    If the ``SMOKE_TEST`` variable is set, no plots are being saved.
+
+    Using the ``BAYBE_MULTIVERSION_PLOTS`` environment variable, it is possible to
+    create plots for the light and dark version of the documentation. If this variable
+    is not set, a single file named ``{base_name}_check.svg`` is created.
+
+    Args:
+        data: The data frame containing the data to be plotted.
+        path: The path of the file that should be saved.
+        base_name: The base name that is used for naming the output files.
+        **kwargs: Keyword arguments. They are directly passed to ``sns.lineplot`` and
+            are used for specifying the plot.
+    """
+    # File containing all the themes
+    themes = json.load(open(Path("plotting_themes.json")))
```

Comment:
Since this was brought up in another PR: Does any of you guys have an idea how to safely open this files independent of the current working directory? The only thing that comes to my mind is via providing yet another argument to `create_plots` (either the path directly or the `themes` dictionary), do you see another solution?
In general, we might want to change this later - just wanted to already ask here in case you see a simple solution I fail to see.

Reply:
UPDATE: I have changed this as follows:
1. If a file "plotting_themes.json" is present in the current folder, use this
2. If no such file is found backtrack from `sys.argv[0]` until you are either in the baybe folder or cannot backtrack further.
3. If you are in the baybe folder, load plotting_themes.json.
4. If not, use a fallback scheme.

This should now be rather safe, as it should work even if there is just no plotting_themes.json, and will only require a minor change if we decide to move the json to another place.

Reply:
`__file__` has the problem that it does not work when we transform the examples to jupyter notebooks, since these do not have a `__file__` attribute. Using `sys` was the easiest way that I found that works reliably for both variants without having to use some crude workarounds.

Reply:
ok got it makes sense